website/src/main.js

```diff
@@ -1,4 +1,6 @@
 /* global document */
+import '@babel/polyfill';
```

Comment:
Does this dependency need to be added to website/package.json?  Or does it work fine with it added at the top level ./package.json?

Reply:
Also curious about the documentation saying it should be added to depenendencies instead of devDependencies: [https://babeljs.io/docs/en/babel-polyfill/](https://babeljs.io/docs/en/babel-polyfill/).  I guess if it worked in our testing..

Reply:
If this is a standalone application it should be added to dependencies. It doesn't matter in the website case since we only distribute the compiled bundle.